packages/devtools/src/connect.ts

```diff
@@ -10,35 +9,38 @@ import {
   switchMap,
 } from "rxjs/operators"
 
-export const createTabState = () => {
-  const reset$ = new Subject<void>()
-
-  const newTag$ = new Subject<{
+export const connect = (input: {
```

Comment:
This function takes in the events received from `rxjs-traces` and processes them into what DevTools use: A list of tags and an event history.

This can be useful for both standalone devtools and the chrome background script that needs to keep an internal state for each tab.

It will probably need revisiting... I'm not really happy with this model of having everything in an Observable of arrays.